source/io/Process.ooc

```diff
@@ -108,7 +108,11 @@ Process: abstract class {
 
 	// Returns a representation of the command, escaped to some point.
 	getCommandLine: func -> String {
-		args join(" ") replaceAll("\\", "\\\\")
+		result := ""
+		for (i in 0 .. args count - 1)
+			result = result << args[i] << " "
+		result = result << args[args count - 1]
```

Comment:
Hm, not sure if we should have a check for that.
`// The first argument should be the path to the executable.`